frontend/app/components/wp-table/wp-table.directive.ts

```diff
@@ -210,7 +223,10 @@ function wpTable(
             setRowSelectionState(row, multipleChecked ? true : !currentRowCheckState);
           }
 
-          openWhenInSplitView(row.object);
+          // Avoid bubbling of elements within the details link
+          if ($event.target.parentElement.className.indexOf('wp-table--details-link') === -1) {
```

Comment:
This was causing old tabs to update as well despite being opened in a new tab. ui-sref correctly ignores clicks with modifiers or `target` set.